test/e2e-v2/cases/mqe/mqe-cases.yaml

```diff
@@ -37,7 +37,7 @@ cases:
     expected: expected/func-OP.yml
 
   # topN-OP-service Global
-  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics exec --expression="top_n(service_sla,3,des)/100"
+  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics exec --expression="top_n(service_resp_time,3,des)"
```

Comment:
@wankai123 sla e2e sorting is not predictable, so, we changed to another metric.